tests/widgets_calendar.rs

```diff
@@ -63,7 +64,7 @@ fn show_month_header() {
     )
     .show_month_header(Style::default());
     let expected = Buffer::with_lines(vec![
-        "     January 2023    ",
+        "    January 2023     ",
```

Comment:
This change is due to changing the implementation to use `Line::render()` instead of manually calculating the position.

When centering text, Ratatui will generally center 1 space to the left if the centering isn't exact instead of 1 space to the right.